vercel.json

```
@@ -656,6 +656,14 @@
       "source": "/platforms/apple/guides/tvos/configuration/integrations/",
       "destination": "/platforms/apple/guides/tvos/integrations/"
     },
+    {
+      "source": "/platforms/perl/enriching-error-data/",
+      "destination": "/platforms/"
+    },
+    {
+      "source": "/platforms/perl/configuration/filtering/",
+      "destination": "/platforms/"
+    },
```

```suggestion
    {
      "source": "/platforms/perl/configuration/filtering/",
      "destination": "/platforms/"
    },
```
This one is already redirecting.